app/src/main/java/org/fossasia/openevent/general/auth/LoginViewModel.kt

```diff
@@ -37,7 +37,8 @@ class LoginViewModel(
 
     fun login(email: String, password: String) {
         if (!isConnected()) return
-        if (hasErrors(email, password)) return
+        if (hasErrors(email, password))
+            return
```

Comment:
Unnecessary changes.

Reply:
I did that for debugging purpose, can please suggest how to revert, do you want me to add a commit with neccesary changes and then submit the pull request.

Reply:
Yes, do changes related to issue only. It is an extra committed file which is not related to the issue. You can drop the commit, revert changes and push again to your local branch. PR will update automatically.